client/lib/signup/test/signup/config/flows.js

```diff
@@ -25,8 +25,6 @@ var flows = {
 };
 
 module.exports = {
-	currentFlowName: 'simple_flow',
```

Comment:
`currentFlowName` / `defaultFlowName` are really bad names, so thanks for working on this.

`defaultFlowName` is currently used to determine which flow is mounted at `/start`, i.e. if it has a value of `main`, the `main` flow will be mounted on `/start` instead of `/start/main`, and users visiting `/start/main` are redirected to `/start`.